baybe/surrogates/base.py

```diff
@@ -253,10 +226,7 @@ def get_available_surrogates() -> list[type[Surrogate]]:
 
 # Register (un-)structure hooks
 # TODO: Needs to be refactored
-converter.register_unstructure_hook(
-    Surrogate,
-    _decode_onnx_str(_block_serialize_custom_architecture(unstructure_base)),
-)
+converter.register_unstructure_hook(Surrogate, _decode_onnx_str(unstructure_base))
```

Comment:
it seems the serialziation block has been completely removed? Is that correct? Why? I guess it has to do with enabling the GP, but that would dieally be done in the GP PR and not here (espeicially without comment why the block was removed). For now I thing `block_serialization_hook` from utils should still be used in this PR

Reply:
Thanks for flagging! Indeed, this must have seemed a confusing to you because I actually forgot to replace the hook with the one that should be used instead (see comment in the code, which I thought the commit is more or less self-explaining). So it's really just a maintenance commit that removes the workaround that was necessary so far. Will add the correct hook and also add a test. Is this clear then or any other concerns? If not, you can resolve once the commit is pushed, will ping you.

Reply:
@Scienfitz: ping